stripe/src/main/java/com/stripe/android/SourceCallback.java

```diff
@@ -0,0 +1,22 @@
+package com.stripe.android;
+
+import com.stripe.android.model.Source;
```

Comment:
This is a quick port of TokenCallback, which is only used for the batteries-included version where you don't control your own threads.